internal/gitserver/gitdomain/exec.go

```diff
@@ -60,6 +63,44 @@ var (
 	}
 )
 
+var gitObjectRe = regexp.MustCompile(`^[a-fA-F\d]*$`)
+
+// isAllowedDiffArg check if diff arg exists as a file. We do some preliminary checks
```

Comment:
nice, thanks for adding that!

typo fix:
```suggestion
// isAllowedDiffArg checks if diff arg exists as a file. We do some preliminary checks
```